circle.yml

```diff
@@ -54,6 +54,11 @@ deployment:
     owner: palantir
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
+  snapshot-oss-jfrog:
+    branch: develop
+    owner: palantir
+    commands:
+      - ./gradlew artifactoryPublish -x test -x findbugsMain -x findbugsTest
```

Comment:
Does `-x check` work? It may disable checkstyle too which is fine. (Should also change bintray to make it faster)